pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools>=61.0", "build", "wheel"]
+
+[project]
+name = "spec_parser"
+version = "0.1.0"
+dependencies = [
+    "Jinja2",
+    "jsonpickle",
+    "rdflib"
+]
+
+[project.scripts]
+spec-parser = "spec_parser.main:main"
+
+[tool.setuptools.package-data]
+"spec_parser.templates.mkdocs" = ["*.j2"]
```

Comment:
is there a way to have everything under `templates` included? Or do we have to add new lines here for each new directory therein? FYI, different generation formats will use different directories (e.g. I already have a `onefile` for when a single file has to be generated, plus some others).